com.modrinth.ModrinthApp.yml

```diff
@@ -0,0 +1,103 @@
+id: com.modrinth.ModrinthApp
+
+runtime: org.gnome.Platform
```

Comment:
why do you need the GNOME platform?

Reply:
similar to other tauri apps, it uses many of it's dependencies - namely libnotify, libsecret, libsoup, and (in the future when tauri is updated and we don't need to bundle a version with an older abi) webkitgtk